content/applications/hr/attendances/management.rst

```diff
@@ -0,0 +1,126 @@
+<attendances/dashboard>`. At the far-right of each record, :icon:`fa-check` :guilabel:`Approve` and
+:icon:`fa-times` :guilabel:`Refuse` buttons appear. Click the corresponding button to either approve
+or refuse the logged attendance record.
+
+.. note::
+   When approving or refusing attendance records from the :guilabel:`Management` dashboard, it is
+   **not** possible to view the details of the attendance record.
+
+Partial approvals
+-----------------
+
+It is possible to approve a portion of the logged :guilabel:`Extra Hours` and not the full amount.
+To approve only a portion of overtime, click into an attendance record to view the details.
+
+Click into the :guilabel:`Extra Hours` field and enter the amount of hours being approved. After
+editing the :guilabel:`Extra Hours field`, the :guilabel:`Worked Extra Hours` field becomes visible
+to display the difference between the :guilabel:`Worked Time` and :guilabel:`Extra Hours` fields.
+
+.. note::
+   The :guilabel:`Worked Time` field is the total hours the employee logged (the :guilabel:`Worked
+   Time` and :guilabel:`Extra Hours` combined). For example, if an employee is scheduled to work
+   eight hours, and worked five extra hours, the :guilabel:`Worked Time` is thirteen hours, and the
+   :guilabel:`Extra Hours` is five hours. The :guilabel:`worked Time` field **cannot** be modified.
+
+Click :icon:`fa-check` :guilabel:`Approve` to approve the updated :guilabel:`Extra Hours`. Once
+partially approved, the :guilabel:`Worked Extra Hours` field disappears, along with the
+:icon:`fa-check` :guilabel:`Approve` button. The difference in hours appears in the :guilabel:`Extra
+Hours` field.
+
+Once overtime hours are partially approved, the status changes to :guilabel:`Approved`.
+
+.. image:: management/record.png
+   :alt: A detailed attendance record.
+
+.. important::
+   Once partial overtime is approved, it is still possible to deny overtime by clicking
+   :icon:`fa-times` :guilabel:`Refuse`. The record can be modified as many times as needed.
+
+.. note::
+   Occasionally, verifying an employee's location when checking-in may be needed. The method the
+   employee checked-in to work is found on the individual record, in the :guilabel:`Mode` field.
+
+   The :guilabel:`Mode` field is not modifiable, as it only log the method the attendance record was
+   created.
+
+   The possible options are:
+
+  - :guilabel:`Kiosk`: the employee physically checked in or out using a :doc:`kiosk <kiosks>`
+    device. The employee was present at the kiosk to check-in.
+  - :guilabel:`Systray`: the employee checked in or out :doc:`directly from the database
+    <check_in_check_out>`. Depending on the :guilabel:`IP Address` displayed on the detailed
+    attendance record, the employee's location at the time of check-in can be determined.
+  - :guilabel:`Manual`: the record was created manually in the **Attendances** app. This is
+    typically done by management, to add a missing record.
```

Comment:
Nitpick- please indent these bullets a space to align with the .. note:: admonition
```suggestion
   - :guilabel:`Kiosk`: the employee physically checked in or out using a :doc:`kiosk <kiosks>`
     device. The employee was present at the kiosk to check-in.
   - :guilabel:`Systray`: the employee checked in or out :doc:`directly from the database
     <check_in_check_out>`. Depending on the :guilabel:`IP Address` displayed on the detailed
     attendance record, the employee's location at the time of check-in can be determined.
   - :guilabel:`Manual`: the record was created manually in the **Attendances** app. This is
     typically done by management, to add a missing record.
```